client/transactions/autocompleter.tsx

```diff
@@ -63,8 +69,9 @@ export default {
 
 		return [ nameOption ];
 	},
-	getOptionLabel( option, query ) {
+	getOptionLabel( option: CompletionOption, query: string ): JSX.Element {
 		const match = computeSuggestionMatch( option.label, query );
```

Comment:
It's possible for `computeSuggestionMatch` to return a null value which would cause issues for what we return here as we try and access properties of `match`. We should consider trying to add a check here for this, but I wasn't sure the best way to do this!

Reply:
I think we're ok with these values being `null`, so I think we can just to optional chaining to fix this:

```ts
// ...
match?.suggestionMatch // etc.
```

I took the liberty of doing this in d6ed25bee2bb17ae80cf9817c2164eaa693badec 🙂